src/App.js

```diff
@@ -1,21 +1,31 @@
 import MockMan from 'mockman-js';
 import { Route, Routes } from "react-router-dom";
 import "./App.css";
+import Archive from './pages/archive/Archive';
 import Login from './pages/auth/Login';
 import Signup from './pages/auth/Signup';
 import Home from "./pages/home/Home";
+import Label from './pages/label/Label';
+import MyNotes from './pages/mynotes/MyNotes';
+import Trash from './pages/trash/Trash';
```

Comment:
can use barrel imports here for simplicity

Reply:
yes will use thanks for suggesting